packages/inngest/src/components/InngestFunction.ts

```diff
@@ -363,6 +363,40 @@ export namespace InngestFunction {
       period: TimeStr;
     };
 
+    /**
+     * Rate limit function runs, only running them a given number of times (limit) per
+     * period. This can optionally include a `key`, which is used to further
+     * constrain throttling similar to idempotency.
+     */
+    throttle?: {
```

Comment:
Does this description need to change? Looks like it's the same as the one for `rateLimit` atm.